File: backend/go/internal/models/group.go
package models

type Group struct {
	ID          uint    `gorm:"primaryKey;column:id" json:"id"`
	UserID      uint    `gorm:"column:user_id" json:"userId"`
	Name        string  `gorm:"column:name" json:"name"`
	Description *string `gorm:"column:description" json:"description"`
	CreatedAt   int64   `gorm:"column:created_at" json:"createdAt"`
	UpdatedAt   int64   `gorm:"column:updated_at" json:"updatedAt"`

	User    User      `gorm:"foreignKey:UserID" json:"user,omitempty"`
	Members []Contact `gorm:"many2many:group_members" json:"contacts"`
}

File: backend/ASPNET/ASPNET/Models/Entities/Group.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ASPNET.Models.Entities
{
    public class Group
    {
        public int Id { get; set; }
        
        [Required]
        public int UserId { get; set; }
        
        [Required]
        [StringLength(255)]
        public string Name { get; set; } = string.Empty;
        
        public string? Description { get; set; }
        
        [Required]
        public long CreatedAt { get; set; }
        
        [Required]
        public long UpdatedAt { get; set; }

        // Navigation properties
        [ForeignKey("UserId")]
        public virtual User User { get; set; } = null!;
        
        public virtual ICollection<Contact> Contacts { get; set; } = new List<Contact>();
    }
}